pandamium_datapack/data/pandamium/functions/triggers/dimension_count.mcfunction

```diff
@@ -0,0 +1,14 @@
+execute store result score <overworld_player_count> variable if entity @a[nbt={Dimension:"minecraft:overworld"}]
+execute store result score <nether_player_count> variable if entity @a[nbt={Dimension:"minecraft:the_nether"}]
+execute store result score <end_player_count> variable if entity @a[nbt={Dimension:"minecraft:the_end"}]
+
+tellraw @s {"text": "==== Dimension Count ====","color":"aqua","bold":true}

```

Comment:
The `=`s should not be bold, 
and I usually put 8 `=`s before and after the title.
Remember to correct the "footer" `=`s once you've done this